apps/backoffice-v2/src/lib/blocks/hooks/useDocumentReviewBlocks/useDocumentReviewBlocks.ts

```diff
@@ -0,0 +1,7 @@
+import { useManualReviewBlocksLogic } from '@/lib/blocks/variants/ManualReviewBlocks/hooks/useManualReviewBlocksLogic/useManualReviewBlocksLogic';
+
+export const useDocumentReviewBlocks = () => {
+  const { blocks } = useManualReviewBlocksLogic();
+
+  return blocks;
+};
```

Comment:
Use `useDocumentBlocks` directly please.

Reply:
To use this hook it would require to duplicate code or perform huge rework on this. I think this can wait for now.

Reply:
How?

Reply:
This is how `useDocumentBlocks` consumption looks like within `useManialReviewBlocksLogic`. Extracting it and its dependencies out of `useManualReviewBlocksLogic` is an overkill in current case. Hiding `useManualReviewBlocksLogic` implementation behind `useDocumentReviewBlocks` should be fine for now. If there will be case to perform solid rework then we going just update implementation of this hook.
<img width="604" alt="image" src="https://github.com/ballerine-io/ballerine/assets/23522869/9a0890d9-e2bf-4e4d-804b-6d41c1aba92e">

Reply:
So you'd like to consume `useDocumentBlocks` directly but at a later time?

Reply:
Yes,if we ever get to this point.